test/IRGen/dllimport.swift

```diff
@@ -1,16 +1,16 @@
-// RUN: %swift -target thumbv7--windows-itanium -emit-ir -parse-as-library -parse-stdlib -module-name dllimport %s -o - -enable-source-import -I %S -swift-version 3 | %FileCheck %s -check-prefix CHECK -check-prefix CHECK-NO-OPT
-// RUN: %swift -target thumbv7--windows-itanium -O -emit-ir -parse-as-library -parse-stdlib -module-name dllimport -primary-file %s -o - -enable-source-import -I %S -swift-version 3 | %FileCheck %s -check-prefix CHECK -check-prefix CHECK-OPT
+// RUN: %swift -target thumbv7--windows-itanium -emit-ir -parse-as-library -parse-stdlib -module-name dllimport %s -o - -enable-source-import -I %S | %FileCheck %s -check-prefix CHECK -check-prefix CHECK-NO-OPT
+// RUN: %swift -target thumbv7--windows-itanium -O -emit-ir -parse-as-library -parse-stdlib -module-name dllimport -primary-file %s -o - -enable-source-import -I %S | %FileCheck %s -check-prefix CHECK -check-prefix CHECK-OPT
 
 // REQUIRES: CODEGENERATOR=ARM
 
 import dllexport
 
 public func get_ci() -> dllexport.c {
-  return dllexport.ci
+  return dllexport.ci.self
```

Comment:
This one looks wrong (well, unnecessary).

Reply:
Yeah, ci is a _property_ not a type. Looks like we just ignore `.self` here, which seems like a bug.

Reply:
No, you're allowed to write `.self` on anything. That's deliberate. But it's not necessary.